lua/blink_luasnip/init.lua

```diff
@@ -21,6 +29,11 @@ end
 local snip_cache = {}
 local doc_cache = {}
 
+local insert_text_mode = {
+  AsIs = 1,
```

Comment:
I don't see this used. If it's an "enum" copied from `blink`, isn't better to "require" it? If it's "private", then just copy the used mode as a constant and a small comment so we know where it comes from.

Reply:
Good catch, I overlooked this part. Blink seems to have implemented only the `AsIs` anyway, see [here](https://github.com/Saghen/blink.cmp/blob/main/LSP_TRACKER.md#item). But they don't seem to be available as enum in their code source so i'll go to a constant.